specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-09-01/examples/CosmosDBDatabaseAccountGet.json

```diff
@@ -79,6 +79,9 @@
           "enableFreeTier": false,
           "apiProperties": {},
           "enableAnalyticalStorage": true,
+          "analyticalStorageConfig": {
```

Comment:
Need to update examples in the following cases as well : List account, CreateMin, CreateMax, Update